generator/src/googleapis/codegen/template_objects.py

```diff
@@ -128,6 +128,36 @@ def __init__(self, def_dict, api, parent=None, wire_name=None,
       self.SetTemplateValue('description',
                             self.ValidateAndSanitizeComment(self.StripHTML(d)))
 
+  def ComputeNonDuplicatedName(self, original_class_name):
+    class_name = original_class_name
+    parent_path = self.parentPath()
+    occurrences = 0
+
+    # increase occurrences as they occur in the path
```

Comment:
We are not increasing the occurrences anymore, can we update the comment to reflect it?

Reply:
Done